Week6/news/src/app/app.component.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  <app-auth></app-auth>
```

Comment:
why are you including auth component here? Shouldn't it be in a separate route?

Reply:
The naming convention was wrong , actually its a header